concreate/descriptor.py

```diff
@@ -44,6 +44,12 @@ def __iter__(self):
     def items(self):
         return self.descriptor.items()
 
+    def label(self, key):
+        for l in self.descriptor['labels']:
```

Comment:
I would prefer label instead of l here its more readable

Reply:
I didn't necessary want to interfere with the method name here.

Reply:
ok - maybe we can come with better naming later.